vpr/src/place/place.cpp

```diff
@@ -148,7 +148,9 @@ static vtr::NdMatrix<float, 2> chany_place_cost_fac({0, 0}); //[0...device_ctx.g
 
 /* The following arrays are used by the try_swap function for speed.   */
 /* [0...cluster_ctx.clb_nlist.nets().size()-1] */
-static vtr::vector<ClusterNetId, t_bb> ts_bb_coord_new, ts_bb_edge_new;
+static vtr::vector<ClusterNetId, t_bb> ts_bb_edge_new, ts_bb_coord_new;
```

Comment:
Comment what each of these is.  ts_ is for cube_bb (used by both 2D architectures and 3D where you can switch layers at switch bloks).
layer_ts is for per layer bounding box. Used with per-layer_bb (useful when you can only switch layers at OPINs).

While a vector of 3D bounding boxes could work in both cases, it slows the single cube_bb code down by 20% or so. Hence we handle the special case of cube_bb with its own (non-vector) data structures for speed so we don't slow down the 2D case.

Comment when ts_layer_sink_pin_count is used.  (only per layer).

Reply:
Also comment ts_nets_to_update.

Reply:
These comments are added to the next PR which I moved the net cost, etc. into another file